### DISCUSSION TITLE
Add Installation Process for Mac to README

### DISCUSSION THREAD

**Discussion by AdonaiVera** [nerfstudio-project/nerfstudio#2399] [General]
Hello Nerf-Studio team,

I recently noticed the deployment of Nerf-Studio V0.3.2 and was thrilled to see that it now supports Mac. However, upon checking the documentation, I couldn't find a detailed installation process specifically for Mac users.

Given the increasing number of Mac users in the developer community, I believe it would be beneficial to have a clear and concise installation guide for Mac in the README. This would not only assist current users but also attract potential users who are on the Mac platform.

I've gathered some preliminary installation steps and would be happy to draft a detailed guide if the team finds it helpful. Here's a brief overview:

Prerequisites: Ensure CUDA is installed (tested with version 11.3).
Environment Setup: Use conda for dependency management with Python >= 3.7.
Dependencies: Install PyTorch with CUDA and tiny-cuda-nn.
Nerf-Studio Installation: Instructions for installing from pip and from the source.
Docker Option: Provide a ready-to-use Docker image for those who prefer Docker setups.
If the team is already working on this or has plans in the pipeline, please let me know. Otherwise, I'd be more than happy to contribute and assist in any way possible.

Thank you for your time and consideration!

Best regards,
Ado

**Comment by aman190202** [comment]
Any detailed documentation?

**Reply by brentyi** [reply]
Not that I'm aware of, but if folks have time to help with a PR that would be appreciated!

**Reply by sedong-hwang** [reply]
Currently looking for documentation for this as well for macOS!

**Reply by brentyi** [reply]
Noted!! Unfortunately this isn't as well-tested as the Linux workflow since nerfstudio isn't as practical without CUDA, but the _expected_ behavior is that things should "just work" if you install PyTorch 2.0 (without CUDA) in a conda/virtual env and then nerfstudio (from source via `pip install -e .` for now). Things might be slow but it's worth a try; if there are install problems you can file an issue on GitHub (or PR with a fix).

**Reply by sedong-hwang** [reply]
I'll give it a shot thanks!

**Reply by sedong-hwang** [reply]
Hi Brent, are you in the discord community for nerfstudio by any chance? If so, could I get the invite link for that? The link on the website doesn't work :/

**Reply by brentyi** [reply]
Yeah! Do neither of these work?
https://discord.com/invite/RyVk6w5WWP
https://discord.com/invite/uMbNqcraFc

**Reply by sedong-hwang** [reply]
Thanks!

**Reply by jabir3936** [reply]
Hello, sorry for bringing up old post but has anyone made the guide yet?

**Reply by feelingsonice** [reply]
Oh man would love this. So it works on the M series Macs? But just slow?

**Reply by feelingsonice** [reply]
Found this https://gist.github.com/shawngraham/9899e77229ddae7ba8068aca083fe0f8
